.github/workflows/docs-staging-delete.yml

```diff
@@ -0,0 +1,65 @@
+name: Docs Staging Deployment. Documentation Review Delete (if exist).
```

Comment:
Other YAML files use two spaces as tabs, while here are four. Can you adjust that to match the other files?